Cargo.toml

```diff
@@ -14,7 +14,7 @@ keywords = ["auth", "authorization", "rbac", "acl", "abac"]
 
 [dependencies]
 regex = "1.3.1"
-rhai = { version = "0.11.1", features = ["sync"] }
+rhai = { version = "0.12.0", features = ["sync"], path = "../rhai" }
```

Comment:
Let's wait rhai@0.12.0 for removing path

Reply:
resolved, no need to use 0.12.0 version in this commit